ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md

```diff
@@ -174,52 +177,72 @@ An example of a snippet of the inventory.yml file is shown below (subset of an a
 
-```yaml
+```yml
+---
 all:
   children:
-    # DC1_Fabric - EVPN Fabric
-    DC1:
+    FABRIC:
       children:
-        DC1_FABRIC:
+        DC1:
           children:
             DC1_SPINES:
               hosts:
-                DC1-SPINE1:
-                  ansible_host: 10.255.0.11
-                DC1-SPINE2:
-                  ansible_host: 10.255.0.12
-            DC1_L3LEAFS:
-              children:
-                DC1_LEAF1:
-                  hosts:
-                    DC1-LEAF1A:
-                      ansible_host: 10.255.0.13
-                    DC1-LEAF1B:
-                      ansible_host: 10.255.0.14
-                DC1_LEAF2:
-                  hosts:
-                    DC1-LEAF2A:
-                      ansible_host: 10.255.0.15
-                    DC1-LEAF2B:
-                      ansible_host: 10.255.0.16
+                dc1-spine1:
+                  ansible_host: 172.16.1.11
+                dc1-spine2:
+                  ansible_host: 172.16.1.12
 ```
 
-Don't confuse ***hosts*** with servers or similar. A host can be anything that can be accessed via SSH or an API, to be managed by Ansible,
+However, going forward we expect that all hostnames specified are reachable without this workaround, hence the entire inventory file looks as follows:
+
+```yml
+---
+all:
+  children:
+    FABRIC:
+      children:
+        DC1:
+          children:
+            DC1_SPINES:
+              hosts:
+                dc1-spine1:
+                dc1-spine2:
+            DC1_L3_LEAVES:
+              hosts:
+                dc1-leaf1a:
+                dc1-leaf1b:
+                dc1-leaf2a:
+                dc1-leaf2b:
+            DC1_L2_LEAVES:
+              hosts:
+                dc1-leaf1c:
+                dc1-leaf2c:
+
+        NETWORK_SERVICES:
+          children:
+            DC1_L3_LEAVES:
+            DC1_L2_LEAVES:
+        CONNECTED_ENDPOINTS:
+          children:
+            DC1_L3_LEAVES:
+            DC1_L2_LEAVES:
+```
+
+Don’t confuse ***hosts*** with servers or similar. A host can be anything that can be accessed via SSH or an API, to be managed by Ansible,
 including Arista switches.
 
 The settings inside the inventory.yml file are defined in a tree-like structure using ***groups***. Groups can contain hosts or other groups -
 making it easier to apply common configuration to a group of devices.
 
-The ***all*** line at the top is a default group that contains all ***hosts*** i.e. all switches. Don't worry too much about that for now.
+The ***all*** line at the top is a default group that contains all ***hosts*** i.e. all switches. Don’t worry too much about that for now.
 
-The ***children:*** keyword is used to define "groups of groups" i.e. just an internal keyword to differentiate between hosts and groups.
+The ***children:*** keyword is used to define “groups of groups” i.e. just an internal keyword to differentiate between hosts and groups.
```

Comment:
Same comment as Tony regarding the change of quotes and double-quotes, better keep the ASCII charcters in there

Reply:
Should be updated, please take another look